lib/LedgerSMB/Upgrade_Tests.pm

```diff
@@ -184,6 +199,18 @@ Human readable, localized display name
 
 has display_name => (is => 'ro', isa => 'Str', required => 1);
 
+=item alignment
+
+Column alignment
```

Comment:
When writing documentation, please make it elaborate enough for another programmer to understand what to expect in the field or even what to put in it (and what the effects of that will/are supposed to be).